src/scripts/constants.py

```diff
@@ -84,3 +86,34 @@
 
 # Types of performance tests currently supported
 TUPLE_SUPPORTED_TESTS = ('lwt-',)
+
+
+# Email creation-related constants below
+
+# Constants for boto3 to connect to AWS Secret Manager
+REGION_NAME = '<REGION_NAME>'
+SECRET_NAME = '<SECRET_NAME>'
+
+# Thresholds for sending email
+THRESH_PERF_REGRESS = 11  # Determined based on retro analysis on Apr 16th, 2023
```

Comment:
I looked at hunter results with changes detected from 2nd March to 16th April, and derived this value (11) such that it would send one email less than not using it. Do you agree with this % value being a suitable threshold or would you suggest a different value?

Reply:
Aha, right. So I didn't realise this was a percentage until further down in the code so you should say that here.